scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -86,6 +87,27 @@ object PubsubJob {
   }
 }
 
+object PubsubInWithAttributesJob {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+    sc.pubsubSubscriptionWithAttributes(args("input"))
+      .map((v: (String, Map[String, String])) => (v._1 + "X", v._2))
+      .saveAsPubsubWithAttributes(args("output"))
```

Comment:
What I mean is
```scala
.map(kv => kv._1 + "X")
.saveAsPubsub()
```
To demonstrate with attributes in but without attributes out, opposite to the other job.